src/styles/grid-container.scss

```diff
@@ -0,0 +1,25 @@
+.container {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(166px, 200px));
+  gap: 46px 48px;
+  justify-content: center;
+}
+
+@media (min-width:488px) {
+  .container {
+    grid-template-columns: repeat(2, 200px);
```

Comment:
it's better or move the card width to a variable and reuse it here as well since your cols width depends on it